docs/flashbots-auction/miners/mev-geth-spec/v04-rpc.md

```diff
@@ -0,0 +1,185 @@
+revertingTxHashes	|Array<`Data`>	|Array of tx hashes within the bundle that are allowed to cause the EVM execution to revert without preventing the bundle inclusion in a block.
+relaySignature	|Array<`Data`>	|An secp256k1 signature signed with an address from the `miner.trustedrelays`. Message signed is a Keccak hash of RLP serialized sequence that contains the following items: array of txs (a sequence of byte arrays representing RLP serialized txs); minTimestamp serialized as an int256, like in the devp2p specification; maxTimestamp serialized as an int256, like in the devp2p specification; revertingTxHashes serialized as an array of byte arrays.
+### Returns
+
+{`boolean`} - `true` if mega bundle has been accepted by the node, otherwise `false`
+
+### Example
+
+```bash
+# Request
+curl -X POST -H 'Content-Type: application/json' --data '{
+    "id": 1337,
+    "jsonrpc": "2.0",
+    "method": "eth_sendMegaBundle",
+    "params": [
+        {
+          "txs" : [
+              "f9014946843b9aca00830493e094a011e5f4ea471ee4341a135bb1a4af368155d7a280b8e40d5f2659000000000000000000000000fdd45a22dd1d606b3782f2119621e928e32743000000000000000000000000000000000000000000000000000000000077359400000000000000000000000000000000000000000000000",
+              "f86e8204d085012a05f200830c350094daf24c20717f428f00d8448d74d67a77f67ceb8287354a6ba7a18000802ea00e411bcb660dd8d47717df89078d2e8160c08e7f11cb7ad0ee935e7436eceb32a013ee00a21b7fa0a9f9c1224d11261648191875d4633aed6003543ea319f12b62"
+          ],
+          "blockNumber" : "0x12ab34",
+          "minTimestamp" : "0x0",
+          "minTimestamp" :"0x0",
+          "revertingTxHashes": [],
+          "relaySignature" :[update docs with a valid sig so people can use it for testing],
+        }
+    ]
+}' <url>
+
+# Response
+{
+    "id": 1337,
+    "jsonrpc": "2.0",
+    "result": "true"
+}
+```
+
+## eth_callBundle
+
+### Description
+
+Simulate a bundle of transactions at the top of a block.
+
+After retrieving the block specified in the `blockNrOrHash` it takes the same `blockhash`, `gasLimit`, `difficulty`, same `timestamp` unless the `blockTimestamp` property is specified, and increases the block number by `1`. `eth_callBundle` will timeout after `5` seconds.
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| txs | `Array<Data>` | Array of signed transactions (`eth_sendRawTransaction` style, signed and RLP-encoded) |
+| blockNumber	| `Quantity` | A hex encoded block number for which this bundle is valid on |
+| stateBlockNumber | `Quantity\|string\|Block Identifier` | Either a hex encoded number or a {Block Identifier} for which state to base this simulation on.
+| timestamp	|`Quantity`	|Block timestamp to be used in replacement of the timestamp taken from the parent block. |
+
+### Returns
+
+Map<`Data`, "error|value" : `Data`> - a mapping from transaction hashes to execution results with error or output (value) for each of the transactions
```

Comment:
Perhaps irrelevant for this PR, but the response data type for `callBundle` is inaccurate. 

For reference: https://docs.flashbots.net/flashbots-auction/searchers/advanced/rpc-endpoint/#eth_callbundle

Reply:
@metachris thanks! definitely relevant :) could you add insert a suggestion for the response?

Reply:
My vote is to split callbundle into its own separate spec. It's not actually a part of the normal mev-geth releases.